### PR TITLE
Feat sanitize filenames

### PR DESCRIPTION
**Pull request by seavan**
sanitize default filenames

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -246,5 +247,7 @@
       "changelog": true
     }
   },
-  "dependencies": {}
+  "dependencies": {
+    "sanitize-filename": "1.6.1"
```
Comment (anri-asaturov) [inline package.json]: If this is a prod dependency, it should go to app/package.json

File: src/helpers/file.js [hunk 2]
```diff
@@ -2,10 +2,11 @@ const electron = require('electron').remote;
 const fs = require('fs');
 const _ = require('lodash');
 const path = require('path');
+const sanitize = require('sanitize-filename');
 
 function requestDownloadPath(fileName) {
     return new Promise((resolve, reject) => {
-        let p = fileName;
+        let p = sanitize(fileName, { replacement: '_' });
```
Comment (anri-asaturov) [inline src/helpers/file.js]: @dchest please review

Comment (dchest) [inline src/helpers/file.js]: Looks good, but it would be nice if it also sanitized when uploading so that there's no surprises with filenames.

Comment (dchest) [inline src/helpers/file.js]: Maybe do this in the SDK?

**Comment by anri-asaturov** [comment]
Ok. Merging this, not sure about 'for upload' let's discuss with product team.

**Pull request merged**